components/application-configuration/api/src/main/resources/ApplicationResources.properties

```diff
@@ -9,4 +9,12 @@ PhenoTips.DBConfigurationClass_livetableColumns=Livetable Columns
 PhenoTips.DBConfigurationClass_livetableSortColumn=Column to sort by default
 PhenoTips.DBConfigurationClass_livetableRows=Number of rows to display by default
 phenotips.DBConfigurationClass.metaInfo=Meta information:
-phenotips.DBConfigurationClass.patientInfo=Patient information:
\ No newline at end of file
+phenotips.DBConfigurationClass.patientInfo=Patient information:
+
+phenotips.LabeledIdentifierConfigurationClass.save=Save changes
```

Comment:
Just 'Save' please.

As a side note, we should find some time to change the other variations in "Administration" ("Save changes", "Save patient form configuration", "Save consents configuration" to "Save", for consistency across all admin sections.